web/html/src/manager/content-management/shared/business/filters.enum.ts

```diff
@@ -116,6 +120,11 @@ const filterMatchers: FilterMatcherEnumType = {
     text: t("provides name"),
     longDescription: t("provides name equal"),
   },
+  PTF_ALL: {
+    key: "ptf_all",
+    text: t("all"),
+    longDescription: t("all product temporary fixes"),
```

Comment:
Are these translations intentionally lower case?

Reply:
All the entries of `FilterMatcherEnumType` have text and description lower case because hey are used within other sentences from what I see.

Reply:
Oh, dang, I see. That's definitely not good because translations won't work correctly that way, but I think it's out of the scope of this PR to fix it. Looking at the file history it's been there for three years give or take. I'll create a spin-off ticket to fix this.

Reply:
Created https://github.com/SUSE/spacewalk/issues/19326